NOTICE

```diff
@@ -1,12 +1,147 @@
-OpenSearch
-Copyright 2021 OpenSearch Contributors
+OpenSearch (https://opensearch.org)
+Copyright OpenSearch Contributors
 
-This product includes software developed by
-Elasticsearch (http://www.elastic.co).
-Copyright 2009-2018 Elasticsearch
+---
+This product includes software developed by Plotly (https://plotly.com).
```

Comment:
Thanks for adding this for third party deps. Curious does all dependencies need to be added? I was checking Dashboards repo which used [query-string](https://github.com/opensearch-project/OpenSearch-dashboards/blob/main/package.json#L194), but I don't see it listed in the [NOTICE file

Reply:
I followed the letter of the definition: "Distribution generally means shipping or allowing download of works" to use on their end.